client/verta/docs/api/api/registry.rst

```diff
@@ -0,0 +1,18 @@
+Model Registry
+==============
+
+Client
+------
+.. automethod:: verta.Client.get_or_create_registered_model
+.. automethod:: verta.Client.get_registered_model
+.. automethod:: verta.Client.get_registered_model_version
+
+Registered Model
+----------------
+.. autoclass:: verta._registry.ModelVersion
+    :members:
+
+Registered Model Version
+------------------------
+.. autoclass:: verta._registry.RegisteredModelVersion
+    :members:
```

Comment:
Sphinx's autodoc extension will fetch the parameters and docstrings for all public methods of the class